packages/kbn-rule-data-utils/src/default_alerts_as_data.ts

```diff
@@ -27,7 +27,7 @@ const ALERT_ACTION_GROUP = `${ALERT_NAMESPACE}.action_group` as const;
 // kibana.alert.case_ids - array of cases associated with the alert
 const ALERT_CASE_IDS = `${ALERT_NAMESPACE}.case_ids` as const;
 
-// kibana.alert.duration.us - alert duration in nanoseconds - updated each execution
+// kibana.alert.duration.us - alert duration in microseconds - updated each execution
```

Comment:
Based on the rule registry [README](https://github.com/elastic/kibana/blob/main/x-pack/plugins/rule_registry/README.md#schema), indeed it is mentioned that `kibana.alert.duration.us` is in microseconds. I just wanted to verify if that's actually true and based on this [test](https://github.com/elastic/kibana/blob/main/x-pack/plugins/rule_registry/common/parse_technical_fields.test.ts), according to the values of the start, @timestamp and duration fields, duration is indeed microseconds.

Reply:
Yes, I can confirm it's in microseconds